src/pretty.rs

```diff
@@ -182,6 +209,18 @@ fn fmt(b: &mut String, limit: isize, mut width: isize, mut docs: Vector<(isize,
                         b.push_str(" ".repeat(indent as usize).as_str());
                         indent as isize
                     }
+                    Mode::Flex => {
+                        if limit - width < broken.len() as isize {
+                            remove_ws_end(b, 1);
+                            b.push_str("\n");
+                            b.push_str(" ".repeat(2).as_str());
```

Comment:
Same indentation situation here, it should respect the current indent level rather than hard coding 2.